README.md

```diff
@@ -21,109 +21,154 @@
 
-Dialogue agents have the potential to transform how humans engage with computers, including faster responses to support queries and free-form interactions with fictional characters, along with many other possibilities. The fastest way to see what `conversant` can do for you is to start talking to a chatbot!
+<!-- Read more about `conversant` on our [launch blog post]() and [threads](), and how it is part of the [Cohere Sandbox](). -->
 
-## Getting Started
+Try `conversant` on our Streamlit demo [here](https://cohere-ai-sandbox-conversant-lib-appstreamlit-example-hgyk4x.streamlitapp.com/). 🎉
 
-Try `conversant` without installing anything [here!](https://cohere-ai-sandbox-conversant-lib-appstreamlit-example-hgyk4x.streamlitapp.com/) :tada:
+## Table of Contents
+1. [Installation and Usage](#installation-and-usage)
+   1. [Installation](#installation)
+   2. [Streamlit Demo](#streamlit-demo)
+   3. [Running Your Own Demo Locally](#running-your-own-demo-locally)
+   4. [Creating a Custom Persona](#creating-a-custom-persona)
+   5. [Editing a Persona on the Demo](#editing-a-persona-on-the-demo)
+   6. [Usage](#usage)
+2. [How Conversant Works](#how-conversant-works)
+3. [Documentation](#documentation)
+4. [Contributing Guidelines](#contributing-guidelines)
+5. [License](#license)
 
-When you launch the app, you'll be able to select a persona from a drop down menu. In this example, we'll be using the Fortune Teller persona.
-
-<center>
-<img src="static/fortune-teller-setup.png" alt="Screenshot showing the description of the Fortune Teller persona, as well as a disclaimer about the demo nature of these personas." width="700">
-</center>
-
-After you've selected a persona, the bot will send a greeting. With each response from the user, the chat progresses.
-
-<center>
-<img src="static/fortune-teller-chat.png" alt="Screenshot showing an exchange between a Fortune Teller chatbot and a user." width="700">
-</center>
+## Installation and Usage
 
 ### Installation
-`conversant` is tested on Python 3.8+ and Cohere 2.8.0+.
 
+`conversant` is available on PyPI, and is tested on Python 3.8+ and [Cohere](https://pypi.org/project/cohere/) 2.4.2+.
 ```
 pip install conversant
 ```
+### Streamlit Demo
 
-We recommend installing `conversant` in a virtual environment. 
+Want to see it in action first? You can use `conversant` on a [Streamlit](https://docs.streamlit.io/) app without installing anything [here!](https://cohere-ai-sandbox-conversant-lib-appstreamlit-example-hgyk4x.streamlitapp.com/) 🎉
 
-### Running a Streamlit Demo
-Cohere uses Streamlit to create its demo applications. If you’re new to Streamlit, you can install it [here](https://docs.streamlit.io/library/get-started/installation) and read more about running Streamlit commands [here](https://docs.streamlit.io/library/get-started/main-concepts).
```

Comment:
I'm inclined to keep details on Streamlit context somewhere -- it was a suggestion from one of our technical writers, @kat-jackson

Reply:
Added back this section under the part about running a local demo

[51cb808](https://github.com/cohere-ai/sandbox-conversant-lib/pull/41/commits/51cb808ffb88454fe97fd89d2d1c8f8bc94e1f85)